### DISCUSSION TITLE
Hdparm.exe trigger alerts in defender why

### DISCUSSION THREAD

**Discussion by SamSepiolWarden** [glpi-project/glpi-agent#776] [Q&A]
### Summary
_
just need help to understand why hdparm.exe is trigerring alerts in defender 
![01114921Caps](https://github.com/user-attachments/assets/db95ac0f-eafc-45be-82d8-eb40081e0970)
it is normal behavior or not ? 
i know hdparm.exe is use by glpi-agent but our controlled folder access trigger alert.
It is normal behavior or not ?

**Comment by g-bougard** [comment]
Hi @SamSepiolWarden 

we don't see your screenshot so I can't tell you if the error is a normal behavior.

Anyway, you should probably ask first Defender support as you still known hdparm.exe is used by glpi-agent.

**Reply by g-bougard** [reply]
It's very probably a false positive.

**Comment by SamSepiolWarden** [comment]
![01114921Caps](https://github.com/user-attachments/assets/3dbe0877-e365-416a-b34d-6d2859a85c94)
the screenshot

**Reply by g-bougard** [reply]
I'm not surprised, this command is a low-level one as it will try to make direct disk access to discover interesting datas like essentially the serialnumber. In the context of glpi-agent it will read datas.

To me, this is really a false positive.

If you don't require storage inventory, you can eventually disable the `storage` category by setting it in `no-category` option. This will prevent the run of hdparm.

**Reply by SamSepiolWarden** [reply]
when you say setting it in no-category for the storage what do you mean ? where i can disable this ? because right now i set up an automatic inventory

**Reply by SamSepiolWarden** [reply]
by the way i deploy my agent via intune

**Reply by g-bougard** [reply]
This is a configuration parameter: https://glpi-agent.readthedocs.io/en/latest/configuration.html#no-category
On windows, MSI installer supports to set it up using `NO_CATEGORY=storage` in your case.
